ahk/window.py

```diff
@@ -508,7 +508,7 @@ def move(self, x='', y='', width=None, height=None):
         )
         self.engine.run_script(script)
 
-    def send(self, keys, delay=None, raw=False, blocking=False, escape=False):
+    def send(self, keys, delay=10, press_duration=-1, raw=False, blocking=False, escape=False):
```

Comment:
This may help avoid a small potential breaking change (if people currently use these arguments positionally)


```suggestion
    def send(self, keys, delay=10, raw=False, blocking=False, escape=False, press_duration=-1):
```